README.md

```diff
@@ -1,5 +1,39 @@
 # Qeb-Hwt
 
-This is the implementation of https://github.com/apps/qeb-hwt
+This is the implementation of https://github.com/apps/qeb-hwt.
 
 ![Thoth Health](https://img.shields.io/badge/Thoth:%20Health-3.5-brightgreen "Thoth Health")
+
+Let Bots, powered with AI and continously learning to increase its knowledge to manage your dependencies.
```

Comment:
I don't know why the sentence is not fitting with me. WDYT about the following suggestions:
```suggestion
Let Bots, powered with AI and continuously learning to increase its knowledge, manage your dependencies.
or 
Let the Bots which are continuously learning to increase its knowledge and are powered with AI, manage your dependencies.
```

Reply:
I like it! Thanks @harshad16 :)